_templates/pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+<!---
+Provide a short summary in the Title above. Examples of good PR titles:
+* "Feature: add so-and-so models"
+* "Fix: deduplicate such-and-such"
+* "Update: dbt version 0.13.0"
+-->
+
+## Description & motivation
+<!---
+Describe your changes, and why you're making them. Is this linked to an open
+issue, a Trello card, or another pull request? Link it here.
+-->
+
+## To-do before merge
+<!---
+(Optional -- remove this section if not needed)
+Include any notes about things that need to happen before this PR is merged, e.g.:
+- [ ] Change the dbt version
+- [ ] Update dbt Cloud jobs
+- [ ] Ensure PR #56 is merged
+-->
+
+## Screenshots:
+<!---
+Include a screenshot of the relevant section of the updated DAG. You can access
+your version of the DAG by running `dbt docs generate && dbt docs serve`.
+-->
+
+## Validation of models:
+<!---
+Include any output that confirms that the models do what is expected. This might
+be a link to an in-development dashboard in your BI tool, or a query that
+compares an existing model with a new one.
+-->
+
+## Changes to existing models:
+<!---
+Include this section if you are changing any existing models. Link any related
+pull requests on your BI tool, or instructions for merge (e.g. whether old
+models should be dropped after merge, or whether a full-refresh run is required)
+-->
+
+## Checklist:
+<!---
+This checklist is mostly useful as a reminder of small things that can easily be
+forgotten – it is meant as a helpful tool rather than hoops to jump through.
+Put an `x` in all the items that apply (while removing the space between the brackets),
+ make notes next to any that haven't beenaddressed, and remove any items that are 
+ not relevant to this PR.
+-->
+- [ ] My pull request represents one logical piece of work.
+- [ ] My commits are related to the pull request and look clean.
+- [ ] My SQL follows the [dbt Labs style guide](https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md).
+- [ ] I have materialized my models appropriately.
+- [ ] I have added appropriate tests and documentation to any new models.
+- [ ] I have updated the README file.
+{%- if project.warehouse == 'redshift' %}
```

Comment:
suggestion: can we get rid of these redshift lines?

Reply:
+1